commands/export.go

```diff
@@ -79,7 +79,9 @@ var ExportJobShowCmd = &cobra.Command{
 }
 
 func init() {
-	ExportCreateCmd.Flags().Bool("attachments", false, "Set to true to include file attachments in the export file.")
+	ExportCreateCmd.Flags().Bool("attachments", true, "Set to true to include file attachments in the export file.")
+	_ = ExportCreateCmd.Flags().MarkHidden("attachments")
+	_ = ExportCreateCmd.Flags().MarkDeprecated("attachments", "the tool now includes attachments by default. The flag will be removed in a future version.")
```

Comment:
nit: we can still have the `--no-attachments` flag, we can check both conditions. Or, we can do that later but since we are here `0/5` adding it.

Like we do in: https://github.com/mattermost/mmctl/pull/370/files

- Hide and deprecate the old flag but still make it functional
- Introduce new flag on the docs

Reply:
That's what I did in the first change. I am not sure how can we support both flags. Which takes higher priority? And why?

I think we should have just a single flag for simplicity. It's okay to change the default behavior. We have done it in the past when we added retries by default for uploads.

0/5 on which flag to add. Let me know.

Reply:
Okay, then let's change the default behavior like you did initially. Using `--attachments=false` sounds odd to me. We will print a deprecation warning anyway. Sorry for the back and forth.

Reply:
Done, PTAL